src/services/near-nep21-util.js

```diff
@@ -370,6 +370,18 @@ export async function swapFromOut(tokenIN, tokenOUT) {
   }
 }
 
+// returns NEAR amount needed for x amount of tokens while adding liquidity
+export async function calcNearAmt(tokenDetails) {
```

Comment:
let's rename the function name, to not make a confusion with near amount for swaps.

How about: `calcNearAddLiquidity`?

Reply:
ok